Samples/README.md

```diff
@@ -2,41 +2,11 @@
-
- Key | Required | Description |
----|---|---
-`issuer` | ✔ | Issuer URL for the client.
-`clientId` | ✔ | Client ID for the Okta application.
-`scopes` | ✔ | Scopes the client is requesting.
-`redirectUri` | ✔  | Redirect URI for the Okta application.
-`logoutRedirectUri` | | Logout URI used for the Okta application.
-Other... | | Any additional keys will be passed to the `additionalParameters` argument of the initializer.
-
-## SingleSignOn (iOS)
-
-This application demonstrates Device SSO authentication (also referred to as `TokenExchangeFlow` within OktaOAuth2). This sample works in conjuction with the WebSignIn (iOS) sample. 
-
-Configure the client by opening the `SingleSignOnViewController.swift` file, and input the client's settings.
-
-If the `Okta.plist` file (from the WebSignIn sample) is configured with the `device_sso` scope, the device secret that is created when authenticating using the web sign in sample is saved to a common keychain entry within a shared App Group.
-
-After signing in, launch the SingleSignOn demo application, and your user should be logged in automatically.
-
-## WebSignIn (macOS)
-
-This is a counterpart to the iOS web sign in sample, and demonstrates a simple macOS SwiftUI application, authenticating over the web.  This application's `Okta.plist` file is shared with the iOS application.
-
-## DeviceAuthSignIn (tvOS)
-
-This sample demonstrates the Device Code Flow (referred to as `DeviceAuthorizationFlow` within OktaOAuth2), and how this can be used to sign in to a tvOS application.
-
-To configure the sample, open the `ViewController.swift` file, and configure the flow with your application's settings.
-
-## UserPasswordSignIn (macOS)
-
-This sample demonstrates the `ResourceOwnerFlow`, which allows users to sign in with a simple username and password. This is a macOS command-line application, and uses command-line arguments to supply client settings.
-
-Simply build the application and, finding the path to the executable within your `DerivedData` folder, and execute the command from the command-line.  Alternatively, you can edit the `UserPasswordSignIn (macOS)` scheme to add arguments, which will enable you to run the sample from within the Xcode debug environment.
+ Sample | Summary |
+ ---|---
+ [WebSignIn Samples](WebSignIn) | Sample applications that demonstrates web-based sign in, and single sign on (SSO).
+ [DeviceAuthSignIn Samples](DeviceAuthSignIn) | Samples demonstrating the Device Authorization Grant flow.
+ [UserPasswordSignIn Samples](UserPasswordSignIn) | Sample demonstrating a CLI app using username/password sign in.
+ [ClassicNativeAuth Sample](ClassicNativeAuth) | Demonstrates integration with Okta's classic auth SDK.
+ [OIDCMigration Sample](OIDCMigration) | Demonstrates migration from the legacy Okta OIDC SDK.
```

Comment:
Put this table in alphabetical order as that what the developer sees. Make each Summary the same as the abstract for that sample. That would make the first couple of rows

[ClassicNativeAuth Sample](ClassicNativeAuth) | Use the Okta Mobile SDK for Swift SDK to add native authentication to your app that uses the classic OktaAuthSdk.
[DeviceAuthSignIn Samples](DeviceAuthSignIn) | Add sign-in using the OAuth 2.0 device authorization grant flow to your tvOS app.